controllers/vcluster_controller.go

```diff
@@ -299,65 +278,29 @@ func (r *VClusterReconciler) redeployIfNeeded(ctx context.Context, vCluster *v1a
 		chartName = constants.DefaultVClusterChartName
 	}
 
-	// chart version
-	var chartVersion string
-	if vCluster.Spec.HelmRelease != nil {
-		chartVersion = vCluster.Spec.HelmRelease.Chart.Version
-	}
-	if chartVersion == "" {
-		chartVersion = constants.DefaultVClusterVersion
```

Comment:
is removing the default version intentional? pretty sure this will break our platform. I don't think hubble specifies the k8s version yet.